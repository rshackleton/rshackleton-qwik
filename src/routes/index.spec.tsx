import { QwikCityMockProvider } from '@builder.io/qwik-city';
import { createDOM } from '@builder.io/qwik/testing';
import { expect, test } from 'vitest';
import IndexRoute from './index';

test(`[Header Component]: Renders without error`, async () => {
  const { screen, render } = await createDOM();
  await render(
    <QwikCityMockProvider>
      <IndexRoute />
    </QwikCityMockProvider>,
  );
  expect(screen.outerHTML).toMatchSnapshot();
});
