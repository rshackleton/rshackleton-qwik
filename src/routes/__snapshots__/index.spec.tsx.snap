// Vitest Snapshot v1

exports[`[Header Component]: Renders without error 1`] = `"<host q:version=\\"0.16.2\\" q:container=\\"resumed\\" q:render=\\"dom-dev\\"><!--qv q:key=WmYC:m6_1 data-qrl=QwikCityMockProvider_component_WmYC5H00wtI--><!--qv q:key q:sref=0 q:s--><!--qv q:key=xYL1:m6_0 data-qrl=routes_component_xYL1qOwPyDI--><!--qv q:key=bfJE:H1_0 data-qrl=Hero_component_bfJE09oKJq8--><section><div class=\\"site-container py-4\\"><h1>Hello World</h1></div></section><!--/qv--><!--/qv--><!--/qv--><!--/qv--></host>"`;
